doc/source/examples/anchor_text_movie.ipynb

```diff
@@ -36,7 +36,16 @@
    "metadata": {
```

Comment:
<p>Remove the warning?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/895/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>